Fix Wording in Updating the View of the Tree (#19)
ietf-wg-keytrans/draft-protocol: draft-ietf-keytrans-protocol.md

tree head is an extension of the previous one. It then also provides the
following:

> - Starting with the log entry that was the oldest in the tree corresponding to
>   the user-advertised size, compute its direct path in the new implicit binary

Reviewer: ```suggestion
- Starting with the rightmost log entry from the tree that was advertised by
  the user, compute its direct path in the new implicit binary
```
